src/beaminteraction/src/4C_beaminteraction_beam_to_solid_mortar_manager.cpp

```diff
@@ -21,6 +22,10 @@
 #include "4C_linalg_fevector.hpp"
 #include "4C_linalg_utils_sparse_algebra_manipulation.hpp"
 #include "4C_linalg_utils_sparse_algebra_math.hpp"
+#include "4C_linalg_vector.hpp"
+#include "4C_mortar_utils.hpp"
```

Comment:
I am fine with this inclusion for this PR, but please open an issue to suggest to move the row-col-transformation utilities out of mortar and to some other, more general place. Originally, they have only been used in mortar, but their functionality is more general and is (at least now) also used for other problems than mortar.

Reply:
See #1067.